docs/how_to/openedx/release_upgrades.md

```diff
@@ -0,0 +1,131 @@
+# How To Update The Release Version Of An Open edX Deployment
+
+## Open edX Releases
+Open edX cuts new named releases approximately every 6 months. Each of our deployments
```

Comment:
link to open edX release schedule documentation?

Reply:
Good call. I'll dig that up and link it.